airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -57,6 +57,9 @@ def download_catalog(catalog_url):
 
 OSS_CATALOG = download_catalog(OSS_CATALOG_URL)
 METADATA_FILE_NAME = "metadata.yaml"
+MANIFEST_FILE_NAME = "manifest.yaml"
+DOCKERFILE_FILE_NAME = "Dockerfile"
+POETRY_FILE_NAME = "pyproject.toml"
 ICON_FILE_NAME = "icon.svg"
```

Comment:
@alafanechere just out of curiosity, do we _actually_ use the icon that is provided with the connector anywhere, or do we require additional pull requests to the platform repo to add an icon? If so, what _should_ we do, long-term?

Reply:
We do use the icon from the connector folder. I thinks it's uploaded on registry generation in the dagster job to feed to [`iconUrl` field](https://connectors.airbyte.com/files/registries/v0/oss_registry.json). This is why we have a static check making sure the file exists.